Build issue with existing projects

Running the docker image on an existing project failed building.
I think this happens because the container is running as root and then the npm command needs the --unsafe-perm param to run a script.

Even after doing this manually the container run without problems but every path will open the  "welcome page". When I try the same without docker is working without a problem.

> api_1  | npm WARN deprecated sprintf@0.1.5: The sprintf package is deprecated in favor of sprintf-js.
api_1  | npm WARN lifecycle my-app@0.1.0\~postinstall: cannot run in wd %s %s (wd=%s) my-app@0.1.0 node node_modules/strapi/lib/utils/post-install.js /usr/src/api/my-app
api_1  | npm notice created a lockfile as package-lock.json. You should commit this file.
api_1  | npm WARN mongoose-float@1.0.2 requires a peer of mongoose@~> 4.x but none is installed. You must install peer dependencies yourself.
api_1  | npm WARN eslint-config-airbnb@13.0.0 requires a peer of eslint-plugin-jsx-a11y@^2.2.3 but none is installed. You must install peer dependencies yourself.
api_1  | 
api_1  | added 509 packages in 21.235s
api_1  | internal/modules/cjs/loader.js:550
api_1  |     throw err;
api_1  |     ^
api_1  | 
api_1  | Error: Cannot find module 'immutable'
api_1  |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:548:15)
api_1  |     at Function.Module._load (internal/modules/cjs/loader.js:475:25)
api_1  |     at Module.require (internal/modules/cjs/loader.js:598:17)
api_1  |     at require (internal/modules/cjs/helpers.js:11:18)
api_1  |     at Object.<anonymous> (/usr/src/api/my-app/plugins/content-type-builder/services/ContentTypeBuilder.js:7:20)
api_1  |     at Module._compile (internal/modules/cjs/loader.js:654:30)
api_1  |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:665:10)
api_1  |     at Module.load (internal/modules/cjs/loader.js:566:32)
api_1  |     at tryModuleLoad (internal/modules/cjs/loader.js:506:12)
api_1  |     at Function.Module._load (internal/modules/cjs/loader.js:498:3)

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,6 +11,8 @@ LABEL maintainer="Luca Perret <perret.luca@gmail.com>" \
 
 WORKDIR /usr/src/api
 
+RUN echo "unsafe-perm = true" >> ~/.npmrc
+
 RUN npm install -g strapi@3.0.0-alpha.14
 
 COPY strapi.sh ./

